terraform/auth0/operations-engineering/clients.tf

```
@@ -431,3 +431,56 @@ resource "auth0_client" "join_github" {
     token_lifetime               = 31557600
   }
 }
+
+resource "auth0_client" "join_github_dev_2" {
+  allowed_clients                       = []
+  allowed_logout_urls                   = ["http://127.0.0.1:4567", "http://127.0.0.1/", "http://localhost:4567", "http://localhost", "http://0.0.0.0", "https://dev.join-github.service.justice.gov.uk/", "http://dev.join-github.service.justice.gov.uk/"]
+  allowed_origins                       = []
+  app_type                              = "regular_web"
+  callbacks                             = ["https://localhost:4567/auth/callback", "http://127.0.0.1:4567/auth/callback", "http://127.0.0.1/auth/callback", "http://localhost:4567/auth/callback", "http://0.0.0.0:4567/auth/callback", "http://localhost/auth/callback", "https://dev.join-github.service.justice.gov.uk/auth/callback", "http://dev.join-github.service.justice.gov.uk/auth/callback"]
+  client_aliases                        = []
+  client_metadata                       = {}
+  cross_origin_auth                     = false
```

```suggestion
  cross_origin_auth                     = true
```
This looks like it's enabled in the test tenant version???